client/src/views/admin/components/Whitelist.vue

```diff
@@ -330,6 +380,26 @@ export default {
       this.adding = false
     },
 
+    onDelete (whitelisted) {
+      this.whitelisted = whitelisted
+      this.deleting = true
+    },
+
+    async remove () {
+      await this.removeFromWhitelist(this.whitelisted._id)
+      this.deleting = false
```

Comment:
Je pense qu'il est plus prudent de réinitialiser this.whitelisted à undefined aprés le remove